src/status_im/ui/screens/communities/reorder_categories.cljs

```diff
@@ -59,20 +66,20 @@
   [{:keys [id name community-id]} is-active? drag]
   (let [background-color (if is-active? colors/gray-lighter colors/white)
         category-none?   (string/blank? id)]
-    [:<>
+    [:<> {:style {:flex 1}}
```

Comment:
`{:style {:flex 1}}` not required, (we no longer have done button, and also it will not work with fragment `[:<>`